boa/src/exec/tests.rs

```diff
@@ -118,6 +118,26 @@ fn object_field_set() {
     assert_eq!(&exec(scenario), "22");
 }
 
+#[test]
+fn object_spread() {
+    let mut context = Context::new();
+
+    let scenario = r#"
+                var b = {x: -1, z: -3}
+                var a = {x: 1, y: 2, ...b};
+            "#;
+    forward(&mut context, scenario);
+
+    let one = forward(&mut context, "a.x");
+    assert_eq!(one, String::from("-1"));
+
+    let two = forward(&mut context, "a.y");
+    assert_eq!(two, String::from("2"));
+
+    let three = forward(&mut context, "a.z");
+    assert_eq!(three, String::from("-3"));
+}
+
```

Comment:
We have a new API for testing simple repetitive cases like this one. You can check it out on:

https://github.com/boa-dev/boa/blob/ae73c261a39f2b3a3d56e8b47d4ca289f56df87c/boa/src/lib.rs#L173-L203

Could you refactor this test to use this API? The other tests are ok.